app/modules/cinema/endpoints_cinema.py

```diff
@@ -28,6 +32,9 @@
 from app.utils.communication.notifications import NotificationTool
 from app.utils.tools import get_file_from_data, save_file_as_data
 
+load_dotenv()
+API_key = getenv("THE_MOVIE_DB_API")
```

Comment:
You can use the class `Settings` and the dependency `get_settings()` to access secrets from the dotenv

Reply:
[Done](https://github.com/aeecleclair/Hyperion/pull/385/commits/47e184ce13bb6b9a873fc7b536b119d55b4189c6)